src/pages/Register.tsx

```
@@ -2,19 +2,57 @@ import { Carousel } from "@material-tailwind/react";
+
+  const handleRegister = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const model: RegisterModel = {
+      fullName,
+      email,
+      password
+    };
+
+
+    try {
+      const response:any = await register({data:{payload:model}}).unwrap();
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)